packages/checkout/sdk/src/smartCheckout/routing/indexer/fetchL1Representation.ts

```diff
@@ -46,9 +47,8 @@ export const fetchL1Representation = async (
     contractAddress: l2address,
   });
 
-  // TODO: When bridge is ready we need to understand how L2 ETH will be mapped back to L1 ETH
```

Comment:
Is this still something we need to look into @dreamoftrees ?

Reply:
All good, I think this is sorted now.